src/custom/pages/Claim/types.ts

```diff
@@ -7,3 +8,11 @@ export type ClaimCommonTypes = {
   tokenCurrencyAmount: CurrencyAmount<Token>
   handleChangeAccount: (e: SyntheticEvent<HTMLButtonElement>) => void
 }
+
+export type ParsedUserClaim = UserClaimData & {
+  parsedAmount: string
+  isFree: boolean
+  currency: string
```

Comment:
If this refers to the investment currency, it could be already part of the `price` if you use `CurrencyAmount<Token>`

Reply:
I'm not sure I understood this one correctly but if our token is vCowToken then the currency will always be vCow, but we need symbol/currency like USDC, GNO, ETH based on the type so we can display it in the UI.